src/parsers.ts

```diff
@@ -3,6 +3,7 @@
  SPDX-License-Identifier: Apache-2.0
 */
 
+import JSON from "json-bigint";
```

Comment:
```suggestion
const JSON = require('json-bigint')({ useNativeBigInt: true });
```

I think we should use the native [`BigInt`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) JS type, by using the above import option.

Without it, the [bignumber.js](https://github.com/MikeMcl/bignumber.js) type is being used. It prevents users from simple comparisons, like native `BitInt` allows (note the lack of String casting):


```js
expect(ce.data).to.equal(1524831183200260097n);
```

Reply:
Thanks for pointing this out. Much nicer.